resource-managers/kubernetes/docker/src/main/dockerfiles/spark-base/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM openjdk:8-alpine
```

Comment:
I tracked down the parentage of this dependency tree and their licenses,
openjdk: MIT
alpine: BSD2
so we are ok.

might need to list them in NOTICE though

Reply:
Seems openjdk is under GPL2. Should they be listed following `This product optionally depends on 'Webbit'` in a similar format and with the license files included under `license`?

Reply:
I was referring to the docker image openjdk
 https://github.com/docker-library/openjdk/blob/b4f29ba829765552239bd18f272fcdaf09eca259/LICENSE

of course, the openjdk bin/bit is not under MIT license. I'm actually not quite sure how "releasing" the docker image goes from the licensing point of view...

Reply:
Got it. Should we defer updating the licensing information later when we have clearer on how we will maintain and release the docker images?

Reply:
we could release with Dockerfile as-is (since the source or lineage doesn't have licensing issue)
we just need to figure out the process of releasing the "convenient" docker images "binaries" on docker hub. there's probably a way to do but let's table that for now.